app/models/ucas_subject.rb

```diff
@@ -2,10 +2,9 @@
 #
 # Table name: ucas_subject
 #
-#  id           :bigint           not null, primary key
-#  type         :text
-#  subject_code :text
+#  id           :integer          not null, primary key
```

Comment:
Is this change caused by a Postgres version mismatch? If it's likely to switch back we should probably avoid including it in this PR to cut down on noise.

Reply:
Also, why is type being taken out?

Reply:
I'm guessing type is being taken out because ucas subjects don't have them, dfe subjects do. Not sure what's going on with the primary key.